assets/src/blocks/ActionsList/index.js

```diff
@@ -62,22 +56,33 @@ export const registerActionsList = () => {
             },
           },
         }],
-        ['core/post-template', {}, [
-          ['core/post-featured-image', {isLink: true}],
-          ['core/group', {}, [
-            ['core/post-terms', {term: 'post_tag', separator: ' '}],
-            ['core/post-title', {isLink: true}],
-            ['core/post-excerpt'],
-          ]],
-          ['core/group', {layout: {type: 'flex', justifyContent: 'right'}}, [
-            ['core/read-more', {
-              className: 'btn btn-small btn-primary',
-              content: __('Take Action', 'planet4-blocks-backend'),
-            }],
+        ['core/group', {}, [
+          ['core/post-template', {}, [
+            ['core/post-featured-image', {isLink: true}],
+            ['core/group', {}, [
+              ['core/post-terms', {term: 'post_tag', separator: ' '}],
+              ['core/post-title', {isLink: true}],
+              ['core/post-excerpt'],
+            ]],
+            ['core/group', {layout: {type: 'flex', justifyContent: 'right'}}, [
+              ['core/read-more', {
+                className: 'btn btn-small btn-primary',
+                content: __('Take Action', 'planet4-blocks-backend'),
```

Comment:
This is not for the scope of this PR, but I've just realised that this should sometimes not be `Take Action` since for Actions we have a sidebar setting that allows editors to change this button text... Not sure how doable it would be to use this here in the Query Loop block

Reply:
I agree to follow up it in another ticket since is a beta implementation. What do you think?

Reply:
Yep let's do that 👍 I've created the [ticket](https://jira.greenpeace.org/browse/PLANET-7369)